packages/medusa/src/loaders/__tests__/feature-flags.spec.ts

```diff
@@ -77,11 +77,11 @@ describe("feature flags", () => {
       join(filesystem.basePath, "flags")
     )
 
-    expect(flags.isFeatureEnabled("flag_1")).toEqual(true)
+    expect(flags.isFeatureEnabled("flag_1")).toEqual(false)
```

Comment:
Q: how come all the expectation are inverted? 😂

Reply:
Wanted to test all cases, check line 100, that test should've failed. But it passed for so long :D